src/utils/currencyData.js

```diff
@@ -0,0 +1,30 @@
+export const currencyData = {
```

Comment:
I believe you would like full name form api ?

Reply:
yeap sure, yesterday we discuss this moment with @dyadenkor